libraries/abstractions/transport/utest/CMakeLists.txt

```diff
@@ -1,101 +1,89 @@
-    list(APPEND mock_list
-                "${kernel_dir}/include/task.h"
-                "${kernel_dir}/include/portable.h"
-                "${common_dir}/include/iot_logging_task.h"
-                "${abstraction_dir}/secure_sockets/include/iot_secure_sockets.h"
-            )
+list(APPEND mock_list
+            "${abstraction_dir}/secure_sockets/include/iot_secure_sockets.h"
+        )
 # list the directories your mocks need
-    list(APPEND mock_include_list
-                "${CMAKE_CURRENT_LIST_DIR}/include"
-            )
+list(APPEND mock_include_list
+            "${CMAKE_CURRENT_LIST_DIR}/include"
+        )
 #list the definitions of your mocks to control what to be included
-    list(APPEND mock_define_list
-                LWIP_COMPAT_SOCKETS=1
-                portHAS_STACK_OVERFLOW_CHECKING=1
-                portUSING_MPU_WRAPPERS=1
-                MPU_WRAPPERS_INCLUDED_FROM_API_FILE
-           )
+list(APPEND mock_define_list
+            LWIP_COMPAT_SOCKETS=1
+            portHAS_STACK_OVERFLOW_CHECKING=1
+            portUSING_MPU_WRAPPERS=1
+            MPU_WRAPPERS_INCLUDED_FROM_API_FILE
+       )
 
 # ================= Create the library under test here (edit) ==================
 
-# The single source of truth for the transport_interface.h is in the coreMQTT repository.
-# In the FreeRTOS Device Software download page in AWS IoT Core, it is possible
-# to choose coreHTTP instead of coreMQTT. In this case, it is necessary to check
-# for the existence of the file in the downloaded software package.
-if( EXISTS ${AFR_MODULES_DIR}/coreMQTT/source )
-    set(transport_interface_dir "${AFR_MODULES_DIR}/coreMQTT/source/portable")
-elseif( EXISTS ${AFR_MODULES_DIR}/coreHTTP/source )
-    set(transport_interface_dir "${AFR_MODULES_DIR}/coreHTTP/source/portable")
-else()
-    message( FATAL_ERROR "No transport_interface.h exists for this included interface.")
-endif()
+# Include filepaths for source and include.
+include( ${AFR_ROOT_DIR}/libraries/abstractions/transport/transport_interface.cmake )
 
 # list the files you would like to test here
-    list(APPEND real_source_files
-                "${AFR_ROOT_DIR}/libraries/abstractions/transport/secure_sockets/transport_secure_sockets.c"
-            )
+list(APPEND real_source_files
+            "${AFR_ROOT_DIR}/libraries/abstractions/transport/secure_sockets/transport_secure_sockets.c"
+        )
 # list the directories the module under test includes
-    list(APPEND real_include_directories
-                .
-                "${transport_interface_dir}"
-                "${AFR_ROOT_DIR}/libraries/abstractions/transport/secure_sockets"
-                "${AFR_ROOT_DIR}/libraries/abstractions/secure_sockets/include"
-                "${AFR_ROOT_DIR}/libraries/c_sdk/standard/common/include"
-            )
+list(APPEND real_include_directories
+            .
+            "${transport_interface_dir}"
+            "${AFR_ROOT_DIR}/libraries/abstractions/transport/secure_sockets"
```

Comment:
This path may not be required as the `transport_interface.h` file does not exist there anymore

Reply:
This is the include for transport_secure_sockets.h, not for transport_interface.h